### ISSUE TITLE
Maybe example with react hooks

### ISSUE THREAD

**Issue by sourcewars** [mojs/mojs#212]
Was somebody able to make this thing work with react and functional components? Would be great if somebody could share some ideas.

**Comment by Sandstedt** [comment]
Good idea. I will see if we can get some more examples overall for usage with different libraries.

I have used it in "old" (pre hooks) React. Then I have declared everything in componentDidMount and if I want the animation in a DOM element I would make a ref and pass it as a parent prop to the Mojs.Shape (or whatever you're using).

**Comment by cctoni** [comment]
I managed to work around the react typescript hooks stuff. Will post the example and for later usage i will refine the typings for mojs and will publish this thingy to npm so you could do @types/mojs in the future

**Comment by xavierfoucrier** [comment]
Related to mojs/mojs#73.

**Comment by xavierfoucrier** [comment]
We will add React example soon to the docs, stay tuned!
